src/hooks/useSearchCurrencyList.ts

```diff
@@ -92,29 +92,29 @@ const useSearchCurrencyList = () => {
           color: colors.yellowFavorite,
           data: abcSort(favoriteAssets, 'name'),
           key: 'favorites',
-          title: lang.t(`exchange.token_sections.${tokenSectionTypes.favoriteTokenSection}`),
+          title: i18n.t((i18n.l.exchange.token_sections as any)[tokenSectionTypes.favoriteTokenSection]),
```

Comment:
Would it work to do something like:

```
title: i18n.t(i18n.l.exchange.token_sections[tokenSectionTypes.favoriteTokenSection as keyof typeof i18n.l.exchange.token_sections]),
```

to avoid any cast

or change the type of `favoriteTokenSection` to be a string union, but that might be a bigger refactor than we want.